usr/share/rear/output/default/94_grub2_rescue.sh

```diff
@@ -104,43 +104,29 @@ if [[ ! -x /etc/grub.d/01_users ]]; then
     chmod 755 /etc/grub.d/01_users
 fi
 
-if [[ $( type -f grub2-mkconfig ) ]]; then
-    grub2-mkconfig -o $grub_conf
-else
-    grub-mkconfig -o $grub_conf
-fi
-
 #Finding UUID of filesystem containing /boot 
 grub_boot_uuid=$(df /boot | awk 'END {print $1}' | xargs blkid -s UUID -o value)
```

Comment:
It is the same line than the your previous comment. In this second commit, I move the `grub-mkconfig `after the creation of a new file named : `/etc/grub.d/45_rear`
```
#!/bin/bash
cat << EOF
menuentry "Relax and Recover" --class os --users "" {
        search --no-floppy --fs-uuid  --set root c3c743c8-23cf-48c5-ac0b-90330dbd0f4e
        linux  /rear-kernel LANG=en_US.UTF-8 SYSFONT=latarcyrheb-sun16 KEYTABLE=us console=hvc0 selinux=0
        initrd /rear-initrd.cgz
        password_pbkdf2 rearadmin grub.pbkdf2.sha512.10000.EBCFA260309BA99CBC309EBC2E6945B920039B1232893EAB33EF7EBC112457B3385913DE35C2CFE72D65215D9C5C672D9C129958BA2F99F7654A99C77C4B98C1.92A76C9CED39F6261D83D7DC96C1784A8E10B0BE5D0456D36E88189CB6CF3DA06AB7C11DAF07E6998F15446849805BFEB7A2B495385F7045B3789B25C00A5E61
}
EOF
```
By doing that, the generation of the rear menuentry will be made automatically by grub. No `sed `postprocess needed. It also avoid the rear menuentry to be removed if a `grub-mkconfig` is run.